static/app/components/onboarding/gettingStartedDoc/onboardingLayout.tsx

```diff
@@ -56,7 +58,8 @@ export function OnboardingLayout({
   const {platformOptions} = docsConfig;
 
   const {introduction, steps, nextSteps} = useMemo(() => {
-    const {onboarding} = docsConfig;
+    const {onboarding, replayOnboarding} = docsConfig;
```

Comment:
This is risky, `docsConfig` is typed as `Docs<any>` which doesn't require `replayOnboarding` to exist.